jest.setup.js

```diff
@@ -1,7 +1,8 @@
 const MockDate = require('mockdate');
 
 // Fix "now" date-time for tests
-MockDate.set('August 1, 2020 10:00:00 UTC');
+// use a local timezone date representation so that tests run on windows
```

Comment:
No need to specify this (would this comment make sense to someone that is part of our discussions?), maybe just updating the existing comment should suffice.
```
// Fix "now" date-time for tests but keeping local timezone